### PR TITLE
#3Task. ivan.lozovskij

### PR DESCRIPTION
**Pull request by Lozovskij**
https://travis-ci.org/github/Lozovskij/db-assignments/builds/715145156

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ before_install:
   - sudo systemctl enable mongod
   - sudo systemctl restart mongod
   - mongod --version
+  - mongorestore dumps/northwind_mongodb/
```
Comment (IlyaMokin) [inline .travis.yml]: why you need the line here?

File: task/mongo-optimization-task.js [hunk 2]
```diff
@@ -241,23 +212,23 @@ async function task_3_1(db) {
...
-            }
+           "$lookup" : {
+               "from" : "clientCriteria",
+                    let: { "opportunities_value": "$criteria_value"},
+                    pipeline: [
+                    {
+                        $match : {
+                            "versions.initiativeId" : ObjectId("58af4da0b310d92314627290")
+                        }
+                    },
```
Comment (IlyaMokin) [inline task/mongo-optimization-task.js]: you can make it on one `$match`. Also, projection and `$limit` can help you here

File: task/mongo-tasks.js [hunk 3]
```diff
@@ -222,10 +222,17 @@ async function task_1_17(db) {
 
 /**
  * Create a query to calcualte total orders count by each day in 1998:
+<<<<<<< HEAD
```
Comment (IlyaMokin) [inline task/mongo-tasks.js]: it looks like you didn't solve merge conflicts

**Comment by Lozovskij** [comment]
https://travis-ci.org/github/AisBrestEDU/db-assignments/builds/727697551

**Comment by vramaniuk** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**